.buildkite/pipeline.yml

```
@@ -22,7 +22,7 @@ steps:
           echo -e "[CUDA_Runtime_jll]\nlocal = \"true\"" >LocalPreferences.toml
 
           # for non-nightly, use Resolver.jl to install the oldest package versions
-          if [[ "{{ matrix.julia }}" != "nightly" ]]; then
+          if [[ "{{ matrix.julia }}" != "nightly" && "{{ matrix.julia }}" != "1.12-nightly" ]]; then
```

```suggestion
          if [[ "{{ matrix.julia }}" != "nightly" && "{{ matrix.julia }}" =~ ^[0-9]+\.[0-9]+$ ]]; then
```

... assuming Bash regexes work here. May need some additional escaping...